content/en/docs/guides/certificate_management/certificate-rotation.md

```diff
@@ -0,0 +1,206 @@
+---
+title: "Root Certificate Rotation"
+description: "Rotating the root certificate in OSM"
+type: docs
+weight: 10
+---
+
+# Root Certificate Rotation
+
+The Mesh's Root certificate is a long-lived certificate that is used by OSM to issue leaf certificates to services in the mesh thereby enabling mTLS. This document focuses on the steps to rotate a certificate, to learn more about OSM's certificates see [Certificate Provider Options](certificates.md).  
```

Comment:
Question, the Mesh's root certificate is meant to be the root certificate that the mesh is using? And not specifically related to MeshRootCertificate?

If so, then I think it should be 
```suggestion
The mesh's root certificate is a long-lived certificate that is used by OSM to issue leaf certificates to services in the mesh thereby enabling mTLS. This document focuses on the steps to rotate a certificate, to learn more about OSM's certificates see [Certificate Provider Options](certificates.md).  
```

Reply:
Also commented on Jackie's comment for this, maybe to make it clearer that it's the root cert, it could be "The root certificate of the mesh"